### DISCUSSION TITLE
hx-swap-oob and delete

### DISCUSSION THREAD

**Discussion by paolino** [bigskysoftware/htmx#2965] [General]
Is there an example where `hx-swap-oob` is used to delete a target?

I tried a plain:
```html
<tbody data-hx-swap-oob="delete:#id-0"></tbody>
```
without success, the element was not removed.

**Comment by aral** [comment]
Just ran into this also. Any thoughts? Were you able to figure this out?

**Comment by aral** [comment]
Right, figured it out based on https://github.com/bigskysoftware/htmx/pull/646/files.

The syntax, using your example, is:

```
<tbody id='id-0' data-hx-swap-oob="delete"></tbody>
```
